### PR TITLE
Revise random beacon skill and agent to get latest block (AEA-1891)

### PR DESCRIPTION
**Pull request by jrriehl**
## Proposed changes

Revise random beacon skill and agent since this is no longer supported by the FetchAI ledger.

Update: rather than deleting skill and agent, just get the latest block data instead of the beacon value.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by 5A11** [comment]
On second thought, let's keep the agent but mark it as "deactivated"/"depreciated" or something similar. Someone might find the code useful.

We can change its name (e.g. add "_(deactivated)" to the relevant folders' name) and explain the situation in the readmes.

File: packages/fetchai/agents/latest_block_feed/aea-config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-agent_name: random_beacon_feed
+agent_name: latest_block_feed
 author: fetchai
 version: 0.10.0
 license: Apache-2.0
-description: An agent that retrieves the Fetch.ai decentralised random beacon value.
+description: An agent that retrieves the latest block data from the Fetch ledger.
```
Comment (5A11) [inline packages/fetchai/agents/latest_block_feed/aea-config.yaml:5]: Need to change the name of the skill below.

**Review by 5A11 — COMMENTED** [review]
Package dependency fails.

**Review by 5A11 — APPROVED** [review]

**Review by solarw — APPROVED** [review]
lgtm

**Pull request merged**